web/viewer.js

```diff
@@ -1074,7 +1081,8 @@ var PDFViewerApplication = {
     document.getElementById('pageNumber').value =
       this.pdfViewer.currentPageNumber;
 
-    this.pdfSidebar.setInitialView(this.preferenceSidebarViewOnLoad);
+    this.pdfSidebar.setInitialView(this.preferenceSidebarViewOnLoad || 
+                                  (sidebarView | 0));
```

Comment:
This line needs one extra space of indenting to align it with the `t` in `this.preferenceSidebarViewOnLoad`.